test/system/156-pull-policy.bats

```diff
@@ -0,0 +1,60 @@
+load helpers
+load helpers.network
+load helpers.registry
+
+function setup() {
+    skip_if_remote "tests depend on start_registry which does not work with podman-remote"
```

Comment:
you skip all test on remote and I see no wiring in of the policy option in the bindings or rest API so I doubt this works on podman-remote.
We need at least one test to ensure the option is working with podman-remote.